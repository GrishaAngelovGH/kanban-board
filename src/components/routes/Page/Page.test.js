import { render } from "@testing-library/react"

import {
  BrowserRouter as Router,
  Routes,
  Route
} from "react-router-dom"

import Page from "./Page"

test("should render routes/Page component", () => {
  const view = render(
    <Router>
      <Routes>
        <Route path="/" element={
          <Page description={<div>Description</div>}>
            <div>content</div>
          </Page>
        } />
      </Routes>
    </Router>
  )

  expect(view).toMatchSnapshot()
})