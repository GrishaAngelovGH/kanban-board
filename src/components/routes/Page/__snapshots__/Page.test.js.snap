// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render routes/Page component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row g-0 vh-100"
        style="background-image: url(geometric-triangle-shapes-background.jpg); background-size: cover;"
      >
        <div
          class="col-12"
        >
          <div
            class="row g-0 bg-light p-2 shadow align-items-center"
          >
            <div
              class="col-2 col-md-1"
            >
              <a
                class="btn btn-light bi bi-arrow-left border"
                href="/"
              />
            </div>
            <div>
              Description
            </div>
          </div>
          <div>
            content
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row g-0 vh-100"
      style="background-image: url(geometric-triangle-shapes-background.jpg); background-size: cover;"
    >
      <div
        class="col-12"
      >
        <div
          class="row g-0 bg-light p-2 shadow align-items-center"
        >
          <div
            class="col-2 col-md-1"
          >
            <a
              class="btn btn-light bi bi-arrow-left border"
              href="/"
            />
          </div>
          <div>
            Description
          </div>
        </div>
        <div>
          content
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
